### ISSUE TITLE
[Accessibility][Screen Readers-Filter Dialog] : Search results of filter dialogs is not accessible with down arrow key.

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2854]
**User Experience** 
If a screen reader will not provide any incorrect way to reach to the search result, the screen reader dependent user will not be able to access the seach result.

**Test Environment**
Browser: Microsoft Edge Dev Version: 84.0.495.2 (Official build)
OS build: 2004 (19562.1000)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE and set the zoom at 400%.
2. Select 'Design Flow' from left navigation.
3. Now, navigate to the filter dialog edit field.
4. Type anything in the search edit field of 'Filter Dialogs'.
5. Observe the narrator narration and try to navigate with down arrow key.
6. Observe the issue.

**Actual:** 
The narrator inform about the search result as "2 results found press down arrow key to navigate" whereas the search results are accessible with tab key instead of down arrow.

**Expected:**
As per narrator narration, the search results should be accessible by down arrow key.

**Note:**
Same issue observed with NVDA.

**MAS Impacted:** [MAS2.1.1](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2})

**Attachment:**

[Search results of filter dialogs is not accessible with down arrow key..zip](https://github.com/microsoft/BotFramework-Composer/files/4562672/Search.results.of.filter.dialogs.is.not.accessible.with.down.arrow.key.zip)
